recipes/ma/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "1.1.1" %}
+{% set sha256 = "a09a8b3d1c6281e2a4c9cddaa13b7ca7e8c600796320df6de3419facaa4d652a" %}
+package:
+  name: ma
+  version: '{{version}}'
+source:
+  url: "https://github.com/ITBE-Lab/MA/archive/{{version}}.tar.gz"
+  sha256: "{{sha256}}"
+build:
+  number: 1
```

Comment:
Since this is the first build, the build number should be set to `0`.

Reply:
Also, only the Mac OS build fails. If this package is not intended for Mac OS, you can skip it by adding `skip: true  # [osx]` to the build section.

Reply:
Thank you!
I have changed the build number and added skip: true # [osx].